src/WfsFilterUtil/WfsFilterUtil.ts

```diff
@@ -142,8 +145,12 @@ class WfsFilterUtil {
         srsName
       };
 
-      if (!_isNil(propertyNames)) {
-        wfsFormatOpts.propertyNames = propertyNames;
+      if (!_isNil(filterPropertyNames)) {
+        wfsFormatOpts.propertyNames = filterPropertyNames;
```

Comment:
Why do we limit the `propertyNames` to the filter-properties? This would also limit the result properties to the properties one has filtered on or would require to set the `propertyNames` prop (see below).

Reply:
Yes, moved it out of `filterProperties`, so it's independently now